File: Crossplatform/c_sharp_version/ServerApp/domain/Agency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    public class Agency
    {
        protected string id;
        protected string agency_name;


        public Agency(string id, string agency_name)
        {
            this.id = id;
            this.agency_name = agency_name;
        }

        public string getAgencyId()
        {
            return id;
        }

        public void setAgencyId(string id)
        {
            this.id = id;
        }

        public string getAgencyName()
        {
            return agency_name;
        }

        public void setAgencyName(string name)
        {
            this.agency_name = name;
        }

        public override string ToString()
        {
            return id + " " + agency_name + '\n';
        }
    }
    }

File: Hibernatfe Java HW/ServerApp/src/main/java/domain/Agency.java
package domain;

import java.util.Objects;

public class Agency {
    protected String id;
    protected String agency_name;

    public Agency(String agency_id, String agency_name) {
        this.id = agency_id;
        this.agency_name = agency_name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getAgency_name() {
        return agency_name;
    }

    @Override
    public String toString() {
        return agency_name + " " + id + '\n';
    }
}
